doc/contributing-guidelines.md

```diff
@@ -0,0 +1,19 @@
+---
+description: "CONTRIBUTORS GUIDE"
+---
```

Comment:
I see you adjusted the single quotes here, and you made it a double quote. Help change it back otherwise, gitbook will not recognize the syntax.

Reply:
@Ifycode done

Reply:
Ok. Thank you @oyenuga17 🥳